launcher/InstanceList.cpp

```diff
@@ -635,8 +635,8 @@ InstancePtr InstanceList::loadInstance(const InstanceId& id)
 
     QString inst_type = instanceSettings->get("InstanceType").toString();
 
-    // NOTE: Some PolyMC versions didn't save the InstanceType properly. We will just bank on the probability that this is probably a OneSix
-    // instance
+    // NOTE: Some PrismMC versions didn't save the InstanceType properly. We will just bank on the probability that this is probably a
```

Comment:
```suggestion
    // NOTE: Some launcher versions didn't save the InstanceType properly. We will just bank on the probability that this is probably a
```
either just launcher or `Prism Launcher`. I think we wanted to leave things like this intentionally generic as to allow people to easily create derivative works (forks)